test/lib.bash

```diff
@@ -45,24 +45,27 @@ function run_e2e_tests {
     && return 1
 }
 
-function run_knative_serving_tests {
-  (
+function checkout_knative_serving {
   local knative_version=$1
   # Setup a temporary GOPATH to safely check out the repository without breaking other things.
   local tmp_gopath
   tmp_gopath="$(mktemp -d -t gopath-XXXXXXXXXX)"
-  cp -r "$GOPATH/bin" "$tmp_gopath"
+  if [[ -d "$GOPATH/bin" ]]; then
+    cp -r "$GOPATH/bin" "$tmp_gopath"
```

Comment:
This cp command works improperly. Bins are directly on `$tmp_gopath` directory. They should be inside `bin/` directory. See, https://github.com/cardil/serverless-operator/blob/feature/rewrite-to-golang/test/e2e/temp_gopath.go#L28-L30

Reply:
Hmm the command at line 54 will copy the whole "bin" directory inside the $tmp_gopath directory. So the path will be `$tmp_gopath/bin/<any_binaries_in_that_dir>` . Your command does the same, no?

Reply:
I think I was mistaken with this. Sorry. When I was doing this those binaries were copied to parent directory.